src/project-2.js

```diff
@@ -1,44 +1,73 @@
 // Do not change any of the function names
 
 const getBiggest = (x, y) => {
+  if (x > y) {
+    return x;
+  }
+  return y;
+
   // x and y are integers.  Return the larger integer
   // if they are the same return either one
 };
 
 const greeting = (language) => {
+  const translations = {
+    German: 'Guten Tag!',
+    Spanish: 'Hola!',
+    Chinese: 'Ni Hao!',
+  };
+  return translations[language] || 'Hello!';
   // return a greeting for three different languages:
   // language: 'German' -> 'Guten Tag!'
   // language: 'Spanish' -> 'Hola!'
   // language: 'Chinese' -> 'Ni Hao!'
   // if language is undefined return 'Hello!'
 };
 
-const isTenOrFive = (num) => {
+const isTenOrFive = num => num === 10 || num === 5;
   // return true if num is 10 or 5
   // otherwise return false
-};
 
-const isInRange = (num) => {
+
+const isInRange = num => num < 50 && num > 20;
   // return true if num is less than 50 and greater than 20
-};
 
-const isInteger = (num) => {
+const isInteger = num => Math.floor(num) === num;
```

Comment:
Creative use of floor.  You can also use `parseInt` to do a similar thing.  It drops everything after the decimal